api/core/project/v1/project.proto

```diff
@@ -43,3 +44,14 @@ message PagerDuty {
   // https://developer.pagerduty.com/api-reference/docs/CONCEPTS.md#services
   repeated string service_ids = 1;
 }
+
+message Link {
+  string name = 1;
+  string url = 2;
+  string image_path = 3; // Can be an s3 location, path on the server, etc.
```

Comment:
i think the group should have an image, not the individual links, or both possibly.